src/translation/translation_de_DE.ts

```diff
@@ -3546,12 +3546,12 @@ Wir haben Deinen Kanal stummgeschaltet und die Funktion &apos;Stummschalten&apos
     <message>
         <location filename="../serverdlg.cpp" line="63"/>
         <source>Or select &apos;%1&apos; and specify a Custom Directory address on the Options tab to register with a custom directory.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Oder wähle &quot;%1&quot; und gebe in der Registerkarte &quot;Optionen&quot; eine benutzerdefinierte Verzeichnisadresse an, um dich bei einem benutzerdefinierten Verzeichnis zu registrieren.</translation>
```

Comment:
`type="unfinished"` should be removed?

Reply:
This still needs review by another translator (I'm not happy with both the translation and the source as it starts with "Or ...". It should give more context)

Reply:
Agreed - hm, probably my fault there:
```java
     QString strDirectoryTypeWT = "<b>" + tr ( "Directory" ) + ":</b> " +
                                  tr ( "Select '%1' not to register your server with a directory." )
                                  .arg ( DirectoryTypeToString     ( AT_NONE ) ) + "<br>" + 
                                  tr ( "Select one of the genres to register with that directory." ) + "<br>" +
                                  tr ( "Or select '%1' and specify a Custom Directory address on the "
                                       "Options tab to register with a custom directory." )
                                      .arg ( DirectoryTypeToString ( AT_CUSTOM ) ) +
```
Should we include the HTML in the English?

Reply:
I think so. Yes.

Reply:
OK, raised #2894 just for this one.

Reply:
Thanks.